test/packages_test.rb

```diff
@@ -36,7 +36,8 @@ def load_zypp(file_name)
 
 PRODUCTS_FROM_ZYPP = load_zypp("products.yml").freeze
 
-describe Yast::Packages do
+describe "Yast::Packages" do
```

Comment:
Why do you need to use the string form?

Reply:
Just because of a better formatting.

It's not strictly needed, but as `Yast::Packages` is an object (not a class) then RSpec prints 
`#<Yast::PackagesClass:0x00000000028047b8>` in the output (see [here](https://travis-ci.org/yast/yast-packager/builds/345128635#L1107)). That does not look nice, printing `Yast::Packages` looks better IMHO.